.example.autotab.yaml

```diff
@@ -1,5 +1,3 @@
-chrome_binary_location: /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
-
 autotab_api_key: ... # Go to https://autotab.com/dashboard to get your API key, or
```

Comment:
add:www browseai.com to the<U> 
BEARSMITH8G © APP A LAN FIBER OPTIC AI CONNECTION NETWORK FOR XFINITY SPEED DATA CONTROL BY JEFFREY AND GE COMPACTRON DOWNTOWN BUFFALO HEADQUARTERS

Reply:
Hello World Read Input Variable Assignment Control Flow Classes
public class Main{

    public static void main(String args[])
    {
        /* Create two objects using constructor; See Employee class at bottom */
        Employee empOne = new Employee("James Smith");
        Employee empTwo = new Employee("Mary Anne");
        
        // Invoking methods for each object created
        empOne.empAge(26);
        empOne.empDesignation("Senior Software Engineer");
        empOne.empSalary(1000);
        empOne.printEmployee();
        
        empTwo.empAge(21);
        empTwo.empDesignation("Software Engineer");
        empTwo.empSalary(500);
        empTwo.printEmployee();
    }
}

class Employee {

    String name;
    int age;
    String designation;
    double salary;

    // This is the constructor of the class Employee
    public Employee(String name) {
        this.name = name;
    }

    // Assign the age of the Employee  to the variable age.
        public void empAge(int empAge) {
        age = empAge;
    }

    /* Assign the designation to the variable designation.*/
        public void empDesignation(String empDesig) {
        designation = empDesig;
    }

    /* Assign the salary to the variable	salary.*/
    public void empSalary(double empSalary) {
        salary = empSalary;
    }

    /* Print the Employee details */
    public void printEmployee() {
        System.out.println("Name:"+ name );
        System.out.println("Age:" + age );
        System.out.println("Designation:" + designation );
        System.out.println("Salary:" + salary);
    }
}